charts/geoserver/values.yaml

```diff
@@ -44,6 +44,10 @@ persistence:
 # the values should be only url without http:// or https://, like "geoserver.example.com"
 whitelist: ""
 
+# the following is for the geoserver java memory heap size control
+envs:
+  extraJavaOpts: "-Xms512m -Xmx2g"
```

Comment:
```suggestion
  EXTRA_JAVA_OPTS: "-Xms512m -Xmx2g"
```

Reply:
made an issue about this in https://github.com/ncsa/charts/issues/17